calblueprint/meepanyar
# Tiffany/customer overview

[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

# What's new in this PR

Updated customer overview & listing to match the Figma, both with styling & functionality. Added a tab component for filtering customers by all/meter/payment/done and updated search feature.

[//]: # "Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

## How to review

- Log in as `testUser`
- Navigate to [http://localhost:3000/customers](http://localhost:3000/customers)
- Click on tabs & search any keywords!

[//]: # 'The order in which to review files and what to expect when testing locally'

## Next steps

- The search bar doesn't properly clear previous searches, e.g. if you type stuff, close, and re-open the search bar, it won't be empty. This doesn't really mess with functionality, just not ideal.
- Total owed numbers still need to be rounded.
- Did we want to keep a sorting feature / search by meter number?
- Couldn't figure out how to get icons next to labels in tabs :(

[//]: # "What's NOT in this PR, doesn't work yet, and/or still needs to be done"

### Screenshots

![image](https://user-images.githubusercontent.com/20329981/114297872-0b317900-9a68-11eb-9c06-e74fb1de1fe5.png)

![image](https://user-images.githubusercontent.com/20329981/114297886-15ec0e00-9a68-11eb-8d8b-a73706d40930.png)

![image](https://user-images.githubusercontent.com/20329981/114297890-20a6a300-9a68-11eb-85d5-4f862c416ef6.png)

![image](https://user-images.githubusercontent.com/20329981/114297894-2603ed80-9a68-11eb-9a0e-75c40213500d.png)

[//]: # "Add screenshots of expected behavior - GIFs if you're feeling fancy!"

CC: @julianrkung

[//]: # 'This tags Julian as a default. Feel free to change, or add on anyone who you should be in on the conversation.'
